deploy/kubeturbo_yamls/turbo-reader.yaml

```diff
@@ -20,6 +21,15 @@ rules:
       - resourcequotas
```

Comment:
As of now kubeturbo does not need `limitranges` in its capabilities.

Reply:
`limitrange` is in the backlog to be supported